apps/desktop/desktop/src/main/resources/css/containers.css

```
@@ -684,7 +684,7 @@
 }
 
 .bisq-card-bg {
-    -fx-background-color: -bisq-dark-grey-40;
+    -fx-background-color: -bisq-dark-grey-20;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)